f5-sdk-dist/Docker/redhat/7/build-rpm.py

```diff
@@ -84,6 +86,40 @@ def make_rpms_build(config):
     return rpm_build
 
 
+def change_requires(buildroot, config):
+    spec = buildroot + "/rpmbuild/SPECS/{}.spec".format(config['project'])
+    try:
+        with open(spec, 'r') as fh:
+            contents = fh.read()
+    except IOError as Error:
+        print("Could not open spec file! ({})".format(Error))
+        raise
+    try:
+        with open(spec, 'w') as fh:
+            for line in contents.split("\n"):
+                if 'Requires' in line:
+                    old = line.replace("Requires: ", "")
+                    Req = namedtuple('Req', 'module, modifier, version')
+                    breakout_re = re.compile('([^<=>]+)([<=>]+)([\d]\S+)')
+                    change = 'Requires: '
+                    modifier_format = "{} {} {}, "
+                    for requirement in old.split(' '):
+                        match = breakout_re.search(requirement)
+                        if match:
+                            req = Req(*match.groups())
+                            mod = 'python-' + req.module \
```

Comment:
Yes, this is the pattern followed by RHEL.  If we are depending on python libs outside of the set of RHEL supported packages, the we have to ask "Do we really need them?"